math/perfect_number.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+
+using namespace std;
+
+void Check_Perfect_Number(int number, int i, int sum) {
+
+  while (i < number) { // to check if the number itself is not less than 1 
```

Comment:
"i" does not need to be incremented until number, it is enough to run until i < number / 2 +1, as number above that are no dividers anyway